rabbitmq/consumer.go

```diff
@@ -0,0 +1,113 @@
+package rabbitmq
+
+import (
+	"github.com/palantir/stacktrace"
+
+	"go.uber.org/zap"
+
+	"github.com/sumup-oss/go-pkgs/logger"
+
+	"github.com/streadway/amqp"
+)
+
+func NewRabbitMQConsumer(
```

Comment:
`NewConsumer`. We're in `RabbitMQ` package anyway.

`rabbitmq.NewConsumer` is sufficient.